### PR TITLE
Ac/tst 1045v2 rebase

### PR DESCRIPTION
**Pull request by achapprazoyo**
#### What?

A description about what this pull request implements and its purpose. Try to be detailed and describe any technical details to simplify the job of the reviewer and the individual on production support.

#### Requirements

- [ ] CHANGELOG.md entry added (required for code changes only)

#### Tickets / Documentation

Add links to any relevant tickets and documentation.

- [Link 1](http://example.com)
- ...

#### Screenshots (if appropriate)

Attach images or add image links here.

Example Image

### CODE REVIEW

File: assets/js/theme/earned-incentives.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                formData.set('action', 'add');
+                formData.set('product_id', e.target.dataset.productId);
+                formData.set('qty[]', '1');
+
+                utils.api.cart.itemAdd(formData, (err, response) => {
+                        console.warn('err', err);
+                        console.warn('err response', response);
+                    if(err == null) {
+                        e.target.innerText = 'Added!';
+                    } else {
```
Comment (andresa-razoyo) [inline assets/js/theme/earned-incentives.js:45]: This empty "else" can be removed

File: assets/js/theme/earned-incentives.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                e.target.innerText = 'Adding...';
+
+                let formData = new FormData();
+                formData.set('action', 'add');
+                formData.set('product_id', e.target.dataset.productId);
+                formData.set('qty[]', '1');
+
+                utils.api.cart.itemAdd(formData, (err, response) => {
+                        console.warn('err', err);
+                        console.warn('err response', response);
```
Comment (andresa-razoyo) [inline assets/js/theme/earned-incentives.js:42]: Are you sure you want these console.warn statements on the production code?

File: assets/js/theme/earned-incentives.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+    async setupIncentives() {
+        await this.grabIncentives();
+        await this.getIncentives(this.cartItemIds);
+    }
+
+    async grabIncentives() {
+        const that = this;
+        const itemIds = [];
+        if(that.context.cartId) {
```
Comment (andresa-razoyo) [inline assets/js/theme/earned-incentives.js:60]: There is a space missing after the "if". Do you have some sort of code formatting tool you can run in your vim/IDE to fix formatting/indentation errors?

File: templates/pages/custom/page/incentives.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{{#partial "page"}}
+    {{> components/common/breadcrumbs breadcrumbs=breadcrumbs}}
+
+    {{! @razoyo todo - fix breadcrumbs }}
+    {{inject "tmp" breadcrumbs }}
+
+    <script>
+        var jsContext = JSON.parse({{jsContext}});
+console.log(jsContext.tmp);
```
Comment (andresa-razoyo) [inline templates/pages/custom/page/incentives.html:9]: This console.log statement should be removed

**Review by razoyodev — APPROVED** [review]

**Pull request merged**